jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/error/filters/DBExecutionErrorFilter.java

```diff
@@ -55,7 +55,7 @@ public boolean accept(ExecutionErrorContext errorContext) {
     
     @Override
     public ExecutionError filter(ExecutionErrorContext errorContext) {
-        Builder errorBuilder = ExecutionError.builder().type("DB");
```

Comment:
Would be good to have constants for the builder types, right now they are hard-coded strings like "DB", "Task". WDYT?